elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/QueryEngine.java

```diff
@@ -85,6 +86,16 @@ public QueryEngine(MetaDataStore metaDataStore) {
         populateMetaData(metaDataStore);
     }
 
+    /**
+     * Construct namespace metadata.
+     *
+     * @param namespacePackage NamespacePackage Type
+     * @return constructed Namespace
+     */
+    protected Namespace constructNamespace(com.yahoo.elide.core.type.Package namespacePackage) {
```

Comment:
This method should abstract like the others.  Let SQLQueryEngine build the actual namespace.